llama_stack/providers/utils/inference/prompt_adapter.py

```diff
@@ -117,27 +116,29 @@ async def _localize_single(c: str | InterleavedContentItem) -> str | RawContentI
         elif isinstance(c, TextContentItem):
             return RawTextItem(text=c.text)
         elif isinstance(c, ImageContentItem):
-            # load image and return PIL version
-            img = c.data
-            if isinstance(img, URL):
-                if img.uri.startswith("data"):
-                    match = re.match(r"data:image/(\w+);base64,(.+)", img.uri)
+            if c.url:
+                # Load image bytes from URL
+                if c.url.uri.startswith("data"):
+                    match = re.match(r"data:image/(\w+);base64,(.+)", c.url.uri)
                     if not match:
                         raise ValueError("Invalid data URL format")
```

Comment:
can you include the first 30-40 chars of the url to show the invalud url?

Reply:
done